Projects/Contracts/Sources/View/CoInsured/CoInusuredInput.swift

```diff
@@ -0,0 +1,273 @@
+import Combine
+import SwiftUI
+import hCore
+import hCoreUI
+
+struct CoInusuredInput: View, KeyboardReadable {
+    @State var fullName: String
```

Comment:
Can we create ViewModel that will contains all of this properties? and switch major of logic there?